Pipfile

```diff
@@ -12,6 +12,7 @@ iterators = "*"
 testcontainers = "*"
 flake8 = "*"
 twine = "*"
+pyyaml = "*"
```

Comment:
Is there a requirements.txt file somewhere that needs updating too?

Reply:
Fixed as well as a requirements for dev